docs/introduction/1-gettingstarted.rst

```diff
@@ -62,9 +62,9 @@ Add more decorated functions to add more routes to your Klein applications.
 Variable Routes
```

Comment:
This file contains references to `codeexamples/template.py` with ``:lines:``. Those line numbers need an update (or switch them to `:pyobject:` if that works with the decorators.

Reply:
Fixed the line numbers, which were already wrong anyway